### PR TITLE
latexmk 4.43a (new formula)

### PR DESCRIPTION
**Pull request by herrbischoff**
Very useful (and required) for `vimtex` for example.

Now in the correct repository and (hopefully) fixed all gotchas pointed out at https://github.com/Homebrew/homebrew/pull/49153.

### CODE REVIEW

File: Formula/latexmk.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  def install
+    mv "latexmk.pl", "latexmk"
+    bin.install "latexmk"
+    man1.install "latexmk.1"
+    share.install "example_rcfiles"
+  end
+
+  def caveats
+    <<-EOS.undent
+    By default latexmk makes dvi files, but not postscript or pdf files. If you generally make pdf files using pdflatex, it would be a good idea to put the following in an appropriate initialization file:
```
Comment (dunn) [inline Formula/latexmk.rb]: If this isn't Homebrew-specific information, let's remove it.

Comment (herrbischoff) [inline Formula/latexmk.rb]: Please compare https://github.com/Homebrew/homebrew/pull/49153#issuecomment-183820674

Comment (dunn) [inline Formula/latexmk.rb]: Oh, I see.  OSX-specific information still isn't quite Homebrew-specific information, but I'll let @DomT4 override me here if he's got an opinion.

Comment (DomT4) [inline Formula/latexmk.rb]: Bearing in mind a significant majority of users ignore caveats with gleeful abandon I lean towards removing it too, or actually supplying a sample initialization file.

File: Formula/latexmk.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  version "4.43a"
+  sha256 "e410d295c0a47327b953ece5b582c294359bdf89138ef990d5621b020ff2bbe5"
+
+  depends_on :tex
+
+  def install
+    mv "latexmk.pl", "latexmk"
+    bin.install "latexmk"
+    man1.install "latexmk.1"
+    share.install "example_rcfiles"
```
Comment (DomT4) [inline Formula/latexmk.rb]: `pkgshare.install` on this please, so we don't dump unnested files in the top-level `share`.

**Comment by DomT4** [comment]
Given how similarly named an [already existing](https://github.com/Homebrew/homebrew-tex/blob/master/Formula/latex-mk.rb) formula is I'm almost tempted to suggest this should be named `latexmk-jcc` or something to be more typo-resistant.

File: Formula/latexmk.rb (hunk 2 above)
Comment (bfontaine) [inline Formula/latexmk.rb]: @herrbischoff ping on this?

**Pull request closed without merging**